tensorflow_addons/losses/sparsemax_loss_test.py

```diff
@@ -130,15 +130,17 @@ def test_sparsemax_loss_positive(dtype):
     assert np.zeros(test_obs).shape == tf_loss_op.shape
 
 
+@pytest.mark.parametrize("dtype", ["float32", "float64"])
+def _tf_sparsemax(z, dtype):
+    tf_sparsemax_op = sparsemax(z.astype(dtype))
+    tf_sparsemax_out = tf_sparsemax_op
+
+    return tf_sparsemax_op, tf_sparsemax_out
```

Comment:
Actually this function is so simple now with eager mode, maybe we could inline (remove) it?

Reply:
Thanks @gabrieldemarmiesse. Updated.